keps/sig-node/2535-ensure-secret-pulled-images/README.md

```diff
@@ -132,10 +131,10 @@ use un-encrypted...
 
 ## Proposal
```

Comment:
I am unclear on how this feature works in the `IfNotPresent` case.

Reply:
if not present will be modified to mean if not present for an authorized use

Reply:
A doc update may be needed.